client/my-sites/domains/domain-management/list/list-all.jsx

```diff
@@ -194,6 +200,9 @@ export default connect(
 		};
 	},
 	( dispatch ) => {
-		return { addDomainClick: () => dispatch( addDomainClick() ) };
```

Comment:
I think we should declare the mapDispatchToProps as an object - it's easier to read and less error prone.